install.sh

```diff
@@ -187,14 +186,27 @@ check_deps() {
 node(){
   # clean up 
   rm -rf node 
+  rm -f SHASUMS256.txt.asc
   rm -rf node-$NODE_VERSION*
-  
+
   echo :Installing Node $NODE_VERSION
-  
+
+  gpg --list-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D 2>&1 >/dev/null || gpg --keyserver pool.sks-keyservers.net --recv-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D
```

Comment:
+1 for https

Does gpg verification make it more secure than simply using https? We are also downloading other binaries over https, so checking this one only might be not very useful.

If gpg checking is really useful, it should be moved to separate function and not break installer on mac.